x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/use_investigate_in_timeline.tsx

```diff
@@ -160,7 +170,7 @@ export const useInvestigateInTimeline = ({
         notes: [],
         timeline: {
           ...timeline,
-          columns: !unifiedComponentsInTimelineDisabled ? defaultUdtHeaders : defaultHeaders,
```

Comment:
so this was only broken by introduction of this feature flag?

Reply:
No it was broken in `8.14` as well and becuase of that this feature flag replaced that broken code and produced same output.

Reply:
Actually, I retract my statement. Yes unified components changes broke it but initial feature flag was introduced in `8.14` and that is why it was broke in `8.14` also.